app/Http/Controllers/Store/RegistrationController.php

```diff
@@ -424,8 +424,7 @@ function ($child) {
 
         // Expect only filled fm variables
         $fm = array_filter(
-            $request->only('fm_privacy'),
-            // Previously: $request->only('fm_chart', 'fm_diary', 'fm_privacy'),
+            $request->all('fm_privacy'),
```

Comment:
Does `all()` take string or this needs to be `[]`?

Reply:
it's cheeky and takes _either_ - there's some code in all() that works out if you asked for a single string attribute or put a bunch in an array.

Reply:
I think `only` did that too. Just checking since somewhere else we use an array with a single item.

Reply:
only changed in 5.5 to NOT return an empty property IF the property was not found.
To maintain 5.4 only('blah') behaviour, you have ti use all('blah').